about.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="author" content="Bogdan Man" />
+    <meta
+      name="description"
+      content="Portofolio for art including photos graphic design and other forms of art"
+    />
+
+    <link rel="stylesheet" href="https://fonts.cdnfonts.com/css/futura-std-4" />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@24,400,0,0"
+    />
+    <link rel="icon" href="./assets/l.png" />
+
+    <link rel="stylesheet" href="stylesheet.css" />
```

Comment:
Puteai separa mai bine CSS-ul. Să fi păstrat părțile comune în `stylesheet.css` și să ai fișiere dedicate sau `<style>` în fiecare fișier.

Reply:
Inainte cand lucram in Angular doar in fisiere separate lucram. Mi-am adus aminte ca pot face asta doar marti la curs.